core/src/main/java/apoc/nodes/Nodes.java

```diff
@@ -83,6 +75,79 @@ public void link(@Name("nodes") List<Node> nodes, @Name("type") String type) {
         }
     }
 
+    @Procedure(mode = Mode.WRITE)
+    @Description("apoc.nodes.deleteAndReconnect([pathLinkedList], [nodesToRemove], {config}) - Removes some nodes from a linked list")
+    public Stream<GraphResult> deleteAndReconnect(@Name("path") Path path, @Name("nodes") List<Node> nodesToRemove, @Name(value = "config", defaultValue = "{}") Map<String,Object> config) {
+
+        DeleteAndReconnectConfig conf = new DeleteAndReconnectConfig(config);
+
+        List<Node> nodeInPath = IterableUtils.toList(path.nodes());
+        List<Relationship> relInPath = IterableUtils.toList(path.relationships());
+
+        List<Node> allMatched = (List<Node>) CollectionUtils.subtract(nodesToRemove, nodeInPath);
```

Comment:
```suggestion
        nodeInPath.removeAll(nodesToRemove)
```

Reply:
Qui ho usato il `substract` e non il `removeAll` perché non volevo effettivamente cancellare gli elementi da nodeInPath, in quanto li utilizzo nel `GraphResult`, ma volevo comunque recuperare gli eventuali nodi non presenti per poi passarli sotto al `throw new RuntimeException(..)` della riga 153.
Oppure potrei usare il `containsAll(..)` e al messaggio di errore dico solo che ci sono nodi che non sono presenti nel path?